Python/omevv/v1/scripts/driftdetection.py

```diff
@@ -0,0 +1,99 @@
+                raise Exception("Error occured while creating manage job ",data);
+        except Exception as e:
+            print(e);
+ 
+if __name__ == "__main__":
+    PARSER = argparse.ArgumentParser(description=__doc__,
+                                     formatter_class=argparse.RawTextHelpFormatter)
+    PARSER.add_argument("--ip", "-i", required=True, help="OME Appliance IP")
+    PARSER.add_argument("--vcusername", "-u", required=True,
+                        help="username of vcenter")
+    PARSER.add_argument("--vcpassword", "-p", required=True,
+                        help="password of vcenter")
+    PARSER.add_argument("--vcUUID", "-d", required=True, default=None,
+                        help="UUID of the relevant vCenter")
+    PARSER.add_argument("--baseline_profile_id", "-b", required=True, default=None, help="baseline profile id of baseline profile for which drift detection schedule needs to be modified")
+    PARSER.add_argument("--monday", "-mon", required=False, default=None, help="indicate the boolean value for monday")
+    PARSER.add_argument("--tuesday", "-tues", required=False, default=None, help="indicate the boolean value for tuesday")
+    PARSER.add_argument("--wednesday", "-wednes", required=False, default=None, help="indicate the boolean value for wednesday")
+    PARSER.add_argument("--thursday", "-thurs", required=False, default=None, help="indicate the boolean value for thursday")
+    PARSER.add_argument("--friday", "-fri", required=False, default=None, help="indicate the boolean value for friday")
+    PARSER.add_argument("--saturday", "-sat", required=False, default=None, help="indicate the boolean value for saturday")
+    PARSER.add_argument("--sunday", "-sun", required=False, default=None, help="indicate the boolean value for sunday")
+    PARSER.add_argument("--time", "-t", required=False, default=None, help="time")
+ 
+    ARGS = PARSER.parse_args()
+
+    if ARGS.ip is not None and ARGS.vcusername is not None and ARGS.vcpassword is not None and ARGS.vcUUID is not None and ARGS.baseline_profile_id is not None:
+        if ARGS.monday is None and ARGS.tuesday is None and ARGS.wednesday is None and ARGS.thursday is None and ARGS.friday is None and ARGS.saturday is None and ARGS.sunday is None and ARGS.time is None:
+            print("No job schedule argument given")
+        else:           
+            base_url = 'https://{ip}/omevv/GatewayService/v1/'.format(ip=ARGS.ip)
+            credential = Credential(username=ARGS.vcusername, password=ARGS.vcpassword)
+            payload = {}
+            driftdetectionhelper = ModifyDriftDetectionScheduleWrapper(base_url=base_url, omeIp=ARGS.ip, vcUsercredential=credential, vCenterUUID=ARGS.vcUUID, payload=payload, baseline_profile_id=ARGS.baseline_profile_id, monday=ARGS.monday, tuesday=ARGS.tuesday, wednesday=ARGS.wednesday, thursday=ARGS.thursday, friday=ARGS.friday, saturday=ARGS.saturday, sunday=ARGS.sunday, time=ARGS.time)
```

Comment:
As a good practice, isnt it good to have all these parameters in create payload rather in class init?